Gemfile.lock

```diff
@@ -232,9 +229,6 @@ GEM
       tzinfo
     event_emitter (0.2.6)
     eventmachine (1.2.5)
-    eventmachine (1.2.5-java)
-    eventmachine (1.2.5-x64-mingw32)
-    eventmachine (1.2.5-x86-mingw32)
```

Comment:
These shouldn't be going away.. we still have an EM dependency from blade

Reply:
Thanks. But `blade` is specified `ruby` for platforms, so Java and Windows unnecessary?
https://github.com/rails/rails/blob/1d24e47140356f136471d15e3ce3fa427f4430c2/Gemfile#L85

Reply:
Ah! :+1: